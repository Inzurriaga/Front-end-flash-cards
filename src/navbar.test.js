import React from 'react';
import Navbar from './NavBar';
import { shallow } from 'enzyme';

describe('nav', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <Navbar />);
  });

  it('should display nav', () => {
    expect(wrapper).toMatchSnapshot();
  });

});