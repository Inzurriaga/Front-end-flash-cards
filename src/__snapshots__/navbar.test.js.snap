// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nav should display nav 1`] = `
<div>
  <div
    className="hamburger-container"
  >
    <div
      className="hamburger-circle"
    />
    <div
      className="stick-container"
    >
      <div
        className="stick stick-one"
      />
      <div
        className="stick stick-two"
      />
      <div
        className="stick stick-three"
      />
    </div>
  </div>
  <ul
    className="nav"
  >
    <li
      onClick={[Function]}
    >
      Main Menu
    </li>
    <li
      onClick={[Function]}
    >
      Study
    </li>
    <li
      onClick={[Function]}
    >
      Quiz
    </li>
    <li
      onClick={[Function]}
    >
      Create
    </li>
    <li
      onClick={[Function]}
    >
      View Decks
    </li>
  </ul>
</div>
`;

exports[`teaList should display all tea cards 1`] = `
<div>
  <div
    className="hamburger-container"
  >
    <div
      className="hamburger-circle"
    />
    <div
      className="stick-container"
    >
      <div
        className="stick stick-one"
      />
      <div
        className="stick stick-two"
      />
      <div
        className="stick stick-three"
      />
    </div>
  </div>
  <ul
    className="nav"
  >
    <li
      onClick={[Function]}
    >
      Main Menu
    </li>
    <li
      onClick={[Function]}
    >
      Study
    </li>
    <li
      onClick={[Function]}
    >
      Quiz
    </li>
    <li
      onClick={[Function]}
    >
      Create
    </li>
    <li
      onClick={[Function]}
    >
      View Decks
    </li>
  </ul>
</div>
`;

exports[`teaList should display nav 1`] = `
<div>
  <div
    className="hamburger-container"
  >
    <div
      className="hamburger-circle"
    />
    <div
      className="stick-container"
    >
      <div
        className="stick stick-one"
      />
      <div
        className="stick stick-two"
      />
      <div
        className="stick stick-three"
      />
    </div>
  </div>
  <ul
    className="nav"
  >
    <li
      onClick={[Function]}
    >
      Main Menu
    </li>
    <li
      onClick={[Function]}
    >
      Study
    </li>
    <li
      onClick={[Function]}
    >
      Quiz
    </li>
    <li
      onClick={[Function]}
    >
      Create
    </li>
    <li
      onClick={[Function]}
    >
      View Decks
    </li>
  </ul>
</div>
`;
